src/nodedef.cpp

```diff
@@ -351,172 +369,225 @@ void ContentFeatures::reset()
 	connects_to.clear();
 	connects_to_ids.clear();
 	connect_sides = 0;
+	color = video::SColor(0xFFFFFFFF);
```

Comment:
Colors are multiplied. This is why white is the default: multiplying with white does nothing.